vue/sbc-common-components/src/components/BrowserVersionAlert.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <sbc-system-banner
+    v-if="browserSupported === false"
+    v-bind:show="browserSupported === false"
+    type="warning"
+    v-bind:message="alertMessage"
+  ></sbc-system-banner>
+</template>
+
+<script lang='ts'>
+import { Component, Vue } from 'vue-property-decorator'
+import SbcSystemBanner from './SbcSystemBanner.vue'
+
+declare module 'vuex' {
+  interface Store<S> {
+    hasModule(_: string[]): boolean
+  }
+}
+
+@Component({
+  components: {
+    SbcSystemBanner
+  },
+  beforeCreate () {
+    this.$store.isModuleRegistered = function (aPath: string[]) {
+      let m = (this as any)._modules.root
+      return aPath.every((p) => {
+        m = m._children[p]
+        return m
+      })
+    }
+  }
+})
+export default class BrowserVersionAlert extends Vue {
+  private alertMessage: string = ''
+  private browserSupported: boolean = true
```

Comment:
Do these props need the `@Prop()` decorator? (The sbc code is a bit different from the Entity UI code so I really don't know if this is needed or helpful here.)

Reply:
Changed